src/pages/collections/_type/_.vue

```diff
@@ -14,6 +14,23 @@
               :is-editorial-description="hasEditorialDescription"
               :title="title"
             />
+            <template
+              v-if="isEditable && userIsEditor"
+            >
+              <div class="d-inline-flex">
+                <b-button
+                  variant="outline-primary"
+                  class="text-decoration-none mr-2"
```

Comment:
I think div doesn't really need a display class here.
The mr-2 class in button is also unnecessary.